### PR TITLE
[8.x] Clarify required software for Homestead - revised

### PR DESCRIPTION
**Pull request by LukeTowers**
Revised according to feedback in https://github.com/laravel/docs/pull/6630#issuecomment-741148852

### CODE REVIEW

**Comment by browner12** [comment]
Is "virtualizers" a commonly used word? I've never seen it used before, that's why I ask.

Is there another term or phrase that is more common? maybe something like "virtualization programs"?

**Comment by LukeTowers** [comment]
@browner12 I'm fine with whatever you want to call it, I just took that term from the VirtualBox about page; didn't really have a better idea of what to call them.

**Comment by browner12** [comment]
learned something new! thanks for pointing that out.

**Comment by driesvints** [comment]
Ping @svpernova09

**Comment by svpernova09** [comment]
These should not be called Virtualizers, Vagrant calls them *Providers*  https://www.vagrantup.com/docs/providers

File: homestead.md [hunk 1]
```diff
@@ -129,7 +129,13 @@ Homestead runs on any Windows, macOS, or Linux system and includes Nginx, PHP, M
 <a name="first-steps"></a>
 ### First Steps
 
-Before launching your Homestead environment, you must install [VirtualBox 6.x](https://www.virtualbox.org/wiki/Downloads), [VMWare](https://www.vmware.com), [Parallels](https://www.parallels.com/products/desktop/), or [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) as well as [Vagrant](https://www.vagrantup.com/downloads.html). All of these software packages provide easy-to-use visual installers for all popular operating systems.
+Before launching your Homestead environment, you must install [Vagrant](https://www.vagrantup.com/downloads.html) as well as one of the following supported virtualizers:
+- [VirtualBox 6.x](https://www.virtualbox.org/wiki/Downloads)
```
Comment (svpernova09) [inline homestead.md:133]: Virtualbox 6.x should be changed to Virtualbox 6.1.x

**Comment by LukeTowers** [comment]
Thanks for the feedback @svpernova09, that's definitely a much better term to use!

**Pull request merged**